tests/test_astronomer_commander.py

```diff
@@ -23,3 +23,28 @@ def test_astronomer_commander_deployment(kube_version):
     assert "quay.io/astronomer/ap-commander:0.25.2" in jmespath.search(
         "spec.template.spec.containers[*].image", doc
     )
+    assert doc["spec"]["template"]["spec"]["containers"][0]["env"][6]["name"] == "COMMANDER_UPGRADE_TIMEOUT"
+    assert doc["spec"]["template"]["spec"]["containers"][0]["env"][6]["value"] == "300"
+
+@pytest.mark.parametrize(
+    "kube_version",
+    supported_k8s_versions,
+)
+def test_astronomer_commander_deployment_upgrade_timeout(kube_version):
+    """Test that helm renders a good deployment template for astronomer/commander."""
+    docs = render_chart(
+        kube_version=kube_version,
+        show_only=["charts/astronomer/templates/commander/commander-deployment.yaml"],
+        values={"commander": {"upgradeTimeout": 600}},
+    )
+
+    assert len(docs) == 1
+    doc = docs[0]
+    assert doc["kind"] == "Deployment"
+    assert doc["apiVersion"] == "apps/v1"
+    assert doc["metadata"]["name"] == "RELEASE-NAME-commander"
+    assert "quay.io/astronomer/ap-commander:0.25.2" in jmespath.search(
+        "spec.template.spec.containers[*].image", doc
+    )
+    assert doc["spec"]["template"]["spec"]["containers"][0]["env"][6]["name"] == "COMMANDER_UPGRADE_TIMEOUT"
+    assert doc["spec"]["template"]["spec"]["containers"][0]["env"][6]["value"] == "300"
```

Comment:
May be I misunderstand this @aliotta - but dont you want to assert it was 600 since that is what you are setting in values above?

Reply:
yeah test was a wip I still can't get the test to pass actually. Going to test locally to see that it works outside the test framework. If so then the bug is in the test framework itself.

Reply:
ah just needed the astronomer prefix should be good now.